### ISSUE TITLE
Add documentation for tests

### ISSUE THREAD

**Issue by rbdavid** [EnzymeFunctionInitiative/EST#96]
The recent PR ( #95, see this [commit](https://github.com/EnzymeFunctionInitiative/EST/pull/95/commits/436972940a70e3f37271ba3229a6c999045993b8)) has added the ability to tell nextflow where files should be written to during the test runs. This and other details should be added to the testing documentation.

**Comment by rbdavid** [comment]
PR #99 includes updated documentation for the new testing functionality. See [docs/source/guides/testing.rst](https://github.com/EnzymeFunctionInitiative/EST/blob/9ff15f087a4f94a37bb43ebe0bd82979757fb0ae/docs/source/guides/testing.rst).

**Comment by nilsoberg** [comment]
@rbdavid Has this been resolved by PR #134?

**Comment by rbdavid** [comment]
Yes, I think this issue should have been closed back in December, but its certainly closed now.
